### ISSUE TITLE
GitLab: one of accounts was blocked because of temporary email. I'll try to fix that.

### ISSUE THREAD

**Issue by liledix4** [liledix4/General-Maintenance#1]
Situation is kinda ridiculous, but it happened. If you open [my @liledix4 page on GitLab](https://gitlab.com/liledix4), you'll see that it's banned. Why? Because I created it using temp email address. *Yeah, there was a period when I cared about privacy to that extent.*

In a way, it makes sense: you can't casually allow everyone to create accounts using temporary email address, keeping in mind that there are so many scammers and abusers.

But did I do something **actually** wrong or harmful? No, absolutely not. There can't be any evidence at all, excluding the fact of using temp email. So, if needed, I'll try to prove my point.

- I still have an access to the temp email that still can receive emails.
    - I was careful about choosing a service that generates disposable email addresses, but at the same time they can be accessed/recovered when needed.
- I still have a password for a banned account.

Basically, I have everything. Now, come on, GitLab Support, do the right job!

**Comment by liledix4** [comment]
So. less than an hour ago, I've opened a ticket on GitLab Support. Now waiting.

Somehow I don't think they'll unban my account. Or not immediately. Well, we'll see.

**Comment by liledix4** [comment]
Now my account is unbanned!
